config/crd/bases/providers.tanzu.vmware.com_templatedefinitions.yaml

```diff
@@ -1,62 +0,0 @@
----
```

Comment:
@anujc25 @tenczar I removed this CRD manifest. Should we keep this or is this okay?